cleanlab/dataset.py

```diff
@@ -113,7 +113,7 @@ def find_overlapping_classes(
     confident_joint=None,
     multi_label=False,
 ):
-    """Returns the classes that are often confused by machine learning model or data labelers.
+    """Returns the classes that are often confused by the data labelers.
```

Comment:
@jwmueller in our language, we should not say `by the data labelers` unless we include other options. I am the one who added ML model here previously. The reason is because not all labels come from labelers. Some labels come from other ML models. other labels come for free (facebook up/down votes or any human-involved data system where human interactions are the labels -- many datasets are of this type)

Reply:
To me, "confused by machine learning model" sounds exactly like confusion matrix.
Even after my attempt at some clarifying statements, I'd still guess many users will still not really understand what is difference between this and confusion matrix.

Reply:
`Returns the pairs of classes that are often mislabeled as one another.`